webapp/components/edit_post_modal.jsx

```diff
@@ -152,6 +152,12 @@ class EditPostModal extends React.Component {
             }
         });
 
+        $(ReactDOM.findDOMNode(this.refs.modal)).on('keydown', (e) => {
```

Comment:
Couple things:
1. If the ref is an HTML element, then you don't need to `ReactDOM.findDOMNode(this.refs.modal)` you can just use `this.refs.modal` (I know you just copied the other ones but they were there from an old React version)
2. Can you turn the event listener off in `componentWillUnmount()`? I know the others aren't being turned off but they should be